corehq/apps/hqwebapp/tests/test_requirejs.py

```diff
@@ -21,6 +21,18 @@ def _run_jobs(cls, files, processor):
             jobs.append(gevent.spawn(processor, filename))
         gevent.joinall(jobs)
 
+    @classmethod
+    def _hqDefine_line(cls, filename):
+        hqDefine_line = None
+        with open(filename, 'r') as f:
+            line = f.readline()
+            while line and not hqDefine_line:
+                if re.match(r'^\s*hqDefine', line):
+                    hqDefine_line = line
+                line = f.readline()
+        return hqDefine_line
```

Comment:
nit: what do you think of using a for-loop here? It also might be slightly more efficient to pre-compile the regex (although I think the re module does keep a cache of the last 100 expressions its seen, so probably negligible):
```py
def _hqDefine_line(cls, filename):
    regex = re.compile(r'^\s*hqDefine')
    with open(filename, 'r') as f:
        for line in f:
            if regex.match(line):
                return line
    return None
```

Reply:
Yes, that's much cleaner, thanks. dd4f9d760ca